### DISCUSSION TITLE
ACSM PDF Can't be Read

### DISCUSSION THREAD

**Discussion by YourLocalMedic** [noDRM/DeDRM_tools#273] [Q&A]
(I'm on Linux) I have downloaded a PDF from a .ascm file with Adobe Digital, and the PDF can be read just fine on there. But when I install the plugin and add the book (after the plugin, not before) it is not readable and still needs a password. Is decrypting this format broken currently? Here is a link to an image of what the book looks like. https://media.discordapp.net/attachments/933897124645572678/1070779131496771594/image.png?width=836&height=470

**Comment by ElleKayEm** [comment]
Nope, not broken.  But it can be tricky on Linux to get the key from ADE into DeDRM.  You might want to take a look at the DeACSM plugin which allows you to skip using ADE and DeDRM can get the key from it instead.

https://www.mobileread.com/forums/showthread.php?t=341975
